pkg/install/install_test.go

```diff
@@ -120,3 +123,62 @@ func TestDeployARMTemplate(t *testing.T) {
 		})
 	}
 }
+
+func TestAddResourceProviderVersion(t *testing.T) {
+
+	ctx := context.Background()
+
+	type test struct {
+		name       string
+		clusterdoc *api.OpenShiftClusterDocument
+		mocks      func(*test, *mock_database.MockOpenShiftClusters)
+		wantErr    string
+	}
+
+	for _, tt := range []*test{
```

Comment:
given that there is just one test (and there is not really a lot of logic to explore later), you could kill the for loop

Reply:
Yeah, that's true. Done.